public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -70,6 +106,9 @@ export const ReceiversTable: FC<Props> = ({ config, alertManagerName }) => {
     [config, grafanaNotifiers.result]
   );
 
+  const errorsByReceiver = (contactPointsState: ContactPointsState, receiverName: string) =>
+    contactPointsState?.receivers[receiverName]?.errorCount ?? 0;
```

Comment:
`contactPointsState` cannot be undefined here
```suggestion
    contactPointsState.receivers[receiverName]?.errorCount ?? 0;
```

Reply:
true